src/ir/possible-constant.h

```diff
@@ -114,8 +114,9 @@ struct PossibleConstantValues {
       auto type = getConstantLiteral().type.getHeapType();
       auto otherType = other.getConstantLiteral().type.getHeapType();
       auto lub = HeapType::getLeastUpperBound(type, otherType);
-      if (lub != type) {
-        value = Literal::makeNull(lub);
+      assert(lub && "TODO: Handle case without LUB");
```

Comment:
I think we should handle this in this PR, or else we'll get fuzzer failures. To handle it, if there is no LUB let's turn the result into `Many()`.

Reply:
Shouldn't we turn it into a `None`, since the types are incompatible? Although once we implement bottom types, we'll be able to make it a `Literal` of that bottom type instead.

Reply:
`None` means "no contents". `Many` means "many things are possible here", which can include many incompatible things.